src/bootstrap.coffee

```diff
@@ -169,7 +169,7 @@ bootstrapOrRetry = ->
 
 hasDeviceApiKeySupport = (osVersion) ->
 	try
-		!/^Resin OS /.test(osVersion) or semver.gte(semverRegex.test(osVersion), '2.0.2')
+		!/^Resin OS /.test(osVersion) or semver.gte(semverRegex.exec(osVersion)[0], '2.0.2')
```

Comment:
Thanks - indeed I have to consider the case when the version has other things appended